.github/workflows/dbld-images.yml

```diff
@@ -0,0 +1,45 @@
+name: Compile dbld-images
+
+on:
+  pull_request:
+  push:
+    paths:
+      - .github/workflows/dbld-images.yml
+      # List is from dbld/rules cache-image-% "WATCHED_FILES"
+      - "dbld/**"
+      - "packaging/rhel/syslog-ng.spec"
+      - "packaging/debian/control"
+  schedule:
+    - cron: '00 03 * * *'
+  workflow_dispatch:
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_CACHE_NAMESPACE: ghcr.io/${{ github.actor }}
```

Comment:
On my fork, this uses `ghcr.io/alltilla`, but I do not have a registry there, and I do not want to create one by accident. Shouldn't we still want `DOCKER_CACHE_NAMESPACE` to be `ghcr.io/syslog-ng`?

Reply:
We agreed that this is somehow the same thing as "do not push on forked repositories". So I removed the env variable. This way dbld will fall back to the default value, which is ghcr.io/syslog-ng.